docs/faq.rst

```diff
@@ -23,7 +23,7 @@ The following steps are specific to VSCode and you can choose to go with it:
 
    .. code-block:: bash
 
-       conda env create -n <your-environment-name> -f conda-env.yml python=3.10
```

Comment:
What is the reason for not having to specify the python version in the command anymore?

Reply:
As we have that constraint (`python=3.10`) in the `conda-env.yml` and also this syntax is incorrect from what I remember.